### ISSUE TITLE
Replace eth-query with ethjs-query?

### ISSUE THREAD

**Issue by mcmire** [MetaMask/core#731]
These packages sound the same but they are different.

* [`eth-query`](https://github.com/ethereumjs/eth-query) is very simple: it makes a JSON-RPC call to the provider and calls the given callback with an error or the resulting data.
* [`ethjs-query`](https://github.com/ethjs/ethjs-query) not only makes a JSON-RPC call to the provider, but it also formats the resulting data according to the spec for the RPC method in question. For instance, if the result is an integer, it converts it from hex format to a `BN` object. Although, see counterpoint: https://consensys.slack.com/archives/GTQAGKY5V/p1634686296360000?thread_ts=1634676086.354900&cid=GTQAGKY5V

Related issue: #108

**Comment by legobeat** [comment]
I guess this is not relevant anymore? #852

**Comment by mcmire** [comment]
I agree that this issue and #852 are mutually exclusive. I made them both because I wasn't sure what our long-term strategy was at the time.

Now that some time has passed, it seems like _this_ issue is definitely relevant: considering that you've recently forked all of the `ethjs-*` packages, it would seem that we at least want to keep using those packages, which would imply that we don't want to use the `eth-*` packages.

As for the issue you mentioned, however: _is_ it irrelevant? Perhaps not everyone would agree with the conclusion I gave — there are arguments to be made for _not_ using Ethers for everything — but even you agree that we shouldn't use the `ethjs-*` packages long-term. But perhaps that view has changed?
